storage/storage_helpers.hpp

```diff
@@ -13,7 +13,7 @@ namespace storage
 class CountryInfoGetter;
 class Storage;
 
-TMwmSize constexpr kMaxMwmSizeBytes = 100 /*Mb*/ * 1024 * 1024;
+TMwmSize constexpr kMaxMwmSizeBytes = 115 /*Mb*/ * 1024 * 1024;
```

Comment:
Yes, it's possible and generally speaking it's better. But I'm not sure it's worth implementing in this PR. This PR is about fixing an issue in IsEnoughSpaceForDownload() which use this const only indirectly.

On the other hand you can have a look at my comment for @Zverik . Anyway it's not accurate method. 

So let's do it in a different PR.